modules/p2-profile/product/pom.xml

```diff
@@ -1293,7 +1293,7 @@
                                 </feature>
 				<feature>
                                     <id>org.wso2.carbon.social.feature.group</id>
-                                    <version>1.1.0</version>
+                                    <version>1.2.0</version>
```

Comment:
This is already fixed in current master branch.
https://github.com/wso2/product-app-manager/blob/master/modules/p2-profile/product/pom.xml#L1255